specification/fluidrelay/resource-manager/Microsoft.FluidRelay/stable/2022-02-15/fluidrelay.json

```diff
@@ -0,0 +1,948 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/FluidRelayServerName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request completed Synchronously."
+          },
+          "204": {
+            "description": "Request completed Synchronously."
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": "Error in operation."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.FluidRelay/fluidRelayServers/{fluidRelayServerName}/regenerateKey": {
+      "post": {
+        "operationId": "FluidRelayServers_RegenerateKey",
+        "summary": "Regenerate the primary or secondary key for this server.",
+        "x-ms-examples": {
+          "Regenerate keys for a Fluid Relay server": {
+            "$ref": "./examples/FluidRelayServers_RegenerateKeys.json"
+          }
+        },
+        "tags": [
+          "FluidRelayServers"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/FluidRelayServerName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RegenerateKeyRequest"
+            },
+            "description": "The details of which keys to generate."
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/FluidRelayServerKeys"
+            },
+            "description": "Request Completed Synchronously."
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": "Error in operation."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.FluidRelay/fluidRelayServers/{fluidRelayServerName}/getKeys": {
+      "post": {
+        "operationId": "FluidRelayServers_GetKeys",
+        "summary": "Regenerate the primary or secondary key for this server.",
```

Comment:
>Regenerate the primary or secondary key for this server. [](http://example.com/codeflow?start=20&length=56)

Fix comment

Reply:
Sorry what do you mean here?

Reply:
The comment says "Regenerate the primary or secondary key for this server." but this is a get keys action

Reply:
Oh got it, will change.

Reply:
Updated.